### PR TITLE
implemented study group members page

### PR DESCRIPTION
**Pull request by Cuiboy**

### CODE REVIEW

**Comment by Cuiboy** [comment]
Screen Shot 2021-02-06 at 3 24 33 PM
Screen Shot 2021-02-06 at 3 24 49 PM

File: berkeley-mobile/StudyPact/StudyGroups/GroupMembersView.swift [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            
+        }
+        return cell
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+    }
+}
+
+/// Cell displaying information for a single study group
```
Comment (huangshawn1) [inline berkeley-mobile/StudyPact/StudyGroups/GroupMembersView.swift:97]: update comment to say "study group member"

**Review by huangshawn1 — COMMENTED** [review]
looks good

**Review by huangshawn1 — APPROVED** [review]
looks good

**Pull request merged**